.github/release-please/config.json

```diff
@@ -45,8 +45,7 @@
     ".": {
       "changelog-path": "CHANGELOG.md",
       "release-type": "rust",
-      "draft": false,
-      "prerelease": false
+      "draft": true
```

Comment:
This parameter will not have the desired affect. 

This means that the release will be a draft and we have to manually update the github release.

Reply:
Are you sure? This is what it shows as a documentation hint:
<img width="510" alt="image" src="https://github.com/user-attachments/assets/1e5ad0c8-114b-4f2c-827b-c2009195ad7d">

Reply:
And this hint says exactly what I am saying: "Create the GitHub release in draft mode."
This does not change the pull request, it changes the GitHub release.

Reply:
I confirmed it in the EDIT above ;) So I attempt to remove also draft, which default is false as we had set. I will try to add the `draft-pull-request: true`, which most probably will work - IDE had it in auto-completion. Fingers crossed. If no change, I will revert it.